tests/test_efs.py

```diff
@@ -101,3 +101,23 @@ def test_delete(self):
         client = factory().client("efs")
         state = client.describe_file_systems().get("FileSystems", [])
         self.assertEqual(state, [])
+
+    def test_kms_alias(self):
+        factory = self.replay_flight_data("test_efs_kms_key_filter")
+        p = self.load_policy(
+            {
+                "name": "efs-kms-alias",
+                "resource": "efs",
+                "filters": [
+                    {
+                        "type": "kms-key",
+                        "key": "c7n:AliasName",
+                        "value": "^(alias/aws/)",
+                        "op": "regex"
+                    }
+                ]
+            },
+            session_factory=factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
one set of minors re tests, we've generally tried to move tests to validate some attribute of the resource instead of just cardinality count alone, to ensure that we're looking at the resource that was expected.

Reply:
Good idea.  I seem to be having trouble getting the SQS Queue URLs to consistently match up with the Queue ARNs.  The URLs taken from ListQueues seems to be applied in random order to the attributes from GetQueueAttributes ?

This makes it difficult to test that both Queues and KmsMasterKeyIds are showing up in the SQS test. (see latest commit)

Reply:
typically its easiest to just sort them explicitly in the test